adoc/chapters/programming_interface.adoc

```diff
@@ -1420,11 +1420,19 @@ info::context::atomic_memory_order_capabilities
 ----
 
     @ [.code]#std::vector<memory_order>#
-   a@ Returns the set of memory orderings supported by atomic operations on all
-      devices in the context, which is guaranteed to include [code]#relaxed#.
+   a@ This query applies only to the capabilities of atomic operations that are
+      applied to memory that can be concurrently accessed by multiple devices
+      in the context.  If these capabilities are not uniform across all devices
+      in the context, the query reports only the capabilities that are common
+      for all devices.
 
-The memory ordering of the context determines the behavior of atomic operations applied
-to any memory that can be concurrently accessed by multiple devices in the context.
+Returns the set of memory orders supported by these atomic operations.  When a
+context returns a "stronger" memory order in this set, it must also return all
```

Comment:
We're using the terms stronger/weaker and wider/narrower, but I don't think we define what these mean, for these to make sense I think we need to define a total ordering for the `memory_order` and `memory_scope`.

Reply:
Good point.  I'll add definitions for "stronger/weaker" to section [3.8.3.1](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_memory_ordering) and definitions for "wider/narrower" to section [3.8.3.2](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#_memory_scope).

Reply:
Addressed in 07393a9249343f061159cbd0b1f4fd16eeda858d.